pkg/sql/catalog/descpb/index.go

```diff
@@ -28,6 +28,11 @@ func (desc *IndexDescriptor) IsPartial() bool {
 	return desc.Predicate != ""
 }
 
+// IsInvisible returns whether the index is invisible or not.
+func (desc *IndexDescriptor) IsInvisible() bool {
+	return desc.Invisible
+}
```

Comment:
Is this method necessary? This isn't super obvious but we try to keep the generated protobuf structs lightweight and instead move as much business logic as possible to the `catalog.Index` interface and its implementation.